catalog/ui/search-ui/standard/src/main/webapp/js/view/preferences/PreferencesModal.view.js

```diff
@@ -320,9 +320,10 @@ define([
             ui: {
                 tbody: 'tbody'
             },
+            viewComparator: 'label',
             initialize: function() {
                 this.collection.each(function(model) {
```

Comment:
It should flow the change up.  

See here for an example of it: http://jsfiddle.net/GarethElms/aQnez/?utm_source=website&utm_medium=embed&utm_campaign=aQnez